contracts/src/IRollup.sol

```diff
@@ -115,6 +117,9 @@ interface IRollup {
     /// @dev Thrown when there are zero stakers
     error NoStaker();
 
+    /// @dev Thrown when the staker amount is set to be increased
+    error IncreaseStake();
```

Comment:
This name sounds misleading to me. Rename it to something that makes it even more easier to understand, for example: **StakeCannotBeIncreased**.